packages/react-components/src/molecules/ToastCard.tsx

```diff
@@ -23,21 +21,52 @@ const iconStyles = css({
   paddingRight: `${15 / perRem}em`,
 });
 
+const emberStyles = css({
+  color: ember.rgb,
+  fill: ember.rgb,
+});
+
+const leadStyles = css({
+  color: lead.rgb,
+  fill: lead.rgb,
+});
+
+type Type = 'alert' | 'attachment' | 'live';
+
+const iconMap: Record<Type, ReactNode> = {
+  alert: alertIcon,
+  attachment: paperClipIcon,
+  live: clockIcon,
+};
+
 interface ToastCardProps {
   readonly children: ReactNode;
-  readonly toastText?: string;
+  readonly toastContent?: string | ReactNode;
+  readonly type?: Type;
 }
-const ToastCard: React.FC<ToastCardProps> = ({ children, toastText }) => (
+const ToastCard: React.FC<ToastCardProps> = ({
```

Comment:
This is growing complex enough that I'd like an own story for it. Also so that the terminology (`live` etc.) is visualised